web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -275,16 +308,52 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
     handleConfirmClose();
   }, [csrf, periods, dispatch, toDelete, handleConfirmClose]);
 
+  const updateReviewPeriodDates = useCallback(
+    async period => {
+      if (!csrf) {
+        return;
+      }
+      const res = await updateReviewPeriod(period, csrf);
+      const data = res?.payload?.data ?? null;
+      if (data) {
+        dispatch({ type: UPDATE_REVIEW_PERIODS, payload: [...periods] });
+      } else {
+        console.log(res?.error);
+        window.snackDispatch({
+          type: UPDATE_TOAST,
+          payload: {
+            severity: 'error',
+            toast: 'Error updating review period'
+          }
+        });
+      }
+      setPeriodToAdd(period);
+    },
+    [csrf, state, periods, dispatch]
+  );
+
   const loadFeedbackTemplates = useCallback(async () => {
     const res = await getAllFeedbackTemplates(csrf);
-    const templates = res.payload.data;
-    templates?.sort((t1, t2) => t1.title.localeCompare(t2.title));
-    setTemplates(templates);
+    const templates = res?.payload?.data;
+    if (templates) {
+      templates?.sort((t1, t2) => t1.title.localeCompare(t2.title));
+      setTemplates(templates);
+    } else {
+      console.log(res?.error);
```

Comment:
Nice! If you also want to keep the console output, I would change it to use `console.error` instead of `console.log`. That will make it appear in red.